samples/ionic/src/app/hivejs/payment/config.ts

```diff
@@ -0,0 +1,118 @@
+    static PASAR_ADDRESS: string = '0x02E8AD0687D583e2F6A7e5b82144025f30e26aA0';
+    static DIAMOND_ADDRESS: string = '0x2C8010Ae4121212F836032973919E8AeC9AEaEE5';
+    static GallERIA_ADDRESS: string = '0xE91F413953A82E15B92Ffb93818d8a7b87C3939B';
+    static CONTRACT_URI = 'https://api.' + Config.BASE_API + '/eth';
+
+    static CONTRACT_CHAINID: number = 20;
+    static CONTRACT_RPC = {
+        20: Config.CONTRACT_URI
+    }
+    /** MainNet IPFS */
+    static IPFS_SERVER: string = 'https://ipfs.trinity-feeds.app/';
+    /** MainNet ASSIST */
+    static ASSIST_SERVER: string = 'https://assist.trinity-feeds.app/';
+    /** TestNet contract */
+    static STICKER_TEST_ADDRESS: string = '0xed1978c53731997f4DAfBA47C9b07957Ef6F3961';
+    static PASAR_TEST_ADDRESS: string = '0x2652d10A5e525959F7120b56f2D7a9cD0f6ee087';
+    static HIVE_NODE_ADDRESS: string = '0x0E7EE480dE19E5BB766bD790774B315b02bfD4e1';
+    static GallERIA_TEST_ADDRESS: string = '0x8b3c7Fc42d0501e0367d29426421D950f45F5041';
+    static DIAMOND_TEST_ADDRESS: string = '';
+
+    static CONTRACT_TEST_URI = 'https://api-testnet.' + Config.BASE_API + '/eth';
+    static CONTRACT_TEST_CHAINID: number = 21;
+    static CONTRACT_TEST_RPC = {
+        21: Config.CONTRACT_TEST_URI
+    }
+    /** TestNet IPFS */
+    static IPFS_TEST_SERVER: string = 'https://ipfs-test.trinity-feeds.app/';
+
+    static changeApi(api: string) {
+        if (api == 'elastos.io') {
+            Config.BASE_API = Config.ELASTOS_API;
+            Config.BRIDGE = Config.ELASTOS_BRIDGE;
+        } else {
+            Config.BASE_API = Config.TRINITY_API;
+            Config.BRIDGE = Config.TRINITY_BRIDGE;
+        }
+
+
+        Config.reset();
+    }
+
+    static reset() {
+        Config.EID_RPC = 'https://api.' + Config.BASE_API + '/eid';
+        Config.CONTRACT_URI = 'https://api.' + Config.BASE_API + '/eth';
+        Config.CONTRACT_RPC = {
+            20: Config.CONTRACT_URI
+        }
+
+        Config.CONTRACT_TEST_URI = 'https://api-testnet.' + Config.BASE_API + '/eth';
+        Config.CONTRACT_TEST_RPC = {
+            21: Config.CONTRACT_TEST_URI
+        }
+    }
+
+    static defaultIPFSApi(): string {
+        const availableIpfsNetworkTemplates: string[] = [
+            "https://ipfs0.trinity-feeds.app/",
+            "https://ipfs1.trinity-feeds.app/",
+            "https://ipfs2.trinity-feeds.app/",
+        ];
+        return availableIpfsNetworkTemplates[Math.floor(Math.random() * availableIpfsNetworkTemplates.length)];
+    }
+
+    static defaultAssistApi(): string {
+        const availableAssistNetworkTemplates: string[] = [
+            "https://assist0.trinity-feeds.app/",
+            "https://assist1.trinity-feeds.app/",
+            "https://assist2.trinity-feeds.app/",
+        ];
+        return availableAssistNetworkTemplates[Math.floor(Math.random() * availableAssistNetworkTemplates.length)];
```

Comment:
A round-robin algorithm would be better to correctly load all servers to the same level.